frontend/src/app/QuickStarts.tsx

```diff
@@ -0,0 +1,56 @@
+import '@patternfly/patternfly/base/patternfly-shield-inheritable.css';
+import '@patternfly/patternfly/utilities/Accessibility/accessibility.css';
```

Comment:
Why do we need additional css here? Should this all be loaded up front?

Reply:
Ah yes we could actually put all this in the same place. The quick start drawers depend on these stylesheets.